app/api/BaseApi.js

```diff
@@ -12,22 +17,22 @@ class BaseApi {
     this.cancelTokenSource = axios.CancelToken.source();
   }
 
-  load = (id) => {
+  load = (id, successCallback, failedCallback) => {
     const options = {
       url: '/api/v1/' + this.responsibleModel + '/' + id + '/' + this.subModel,
       method: 'GET',
       cancelToken: this.cancelTokenSource.token,
     };
 
-    return BaseApi.request(options);
+    BaseApi.sendRequest(options, successCallback, failedCallback);
   }
 
   cancelRequest = () => {
     this.cancelTokenSource.cancel();
   }
 
-  static request = async (options, endpoint) => {
-    const endpointUrl = endpoint || await AsyncStorage.getItem('ENDPOINT_URL');
+  static request = async (options, token = '') => {
```

Comment:
@limkimsan I would suggest to modify this method to accept s.th like this, static request = async (url, options, token = null_or_nil, contentType = 'json')

where the url shall generate by the 

UrlUtil.getAbsoluteUrl(relativeUrl) {
    const endpointUrl = await AsyncStorage.getItem('ENDPOINT_URL') || environment.domain;
    return endpointUrl + relativeUrl;
}